src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -185,6 +189,35 @@ private function extractFromAccessor(string $class, string $property): ?array
         return null;
     }
 
+    /**
+     * Tries to extract type information from constructor.
+     *
+     * @return Type[]|null
+     */
+    private function extractFromConstructor(string $class, string $property): ?array
+    {
+        try {
+            $class = new \ReflectionClass($class);
```

Comment:
Instead of this:

```php
$class = new \ReflectionClass($class);
$constructor = $class->getConstructor();
```

maybe we can use this:

```php
$constructor = (new \ReflectionClass($class))->getConstructor();
```

Reply:
fixed.